app/src/main/java/org/schabi/newpipe/util/StreamDialogEntry.java

```diff
@@ -40,8 +41,14 @@
 
     append_playlist(R.string.append_playlist, (fragment, item) -> {
         if (fragment.getFragmentManager() != null) {
-            PlaylistAppendDialog.fromStreamInfoItems(Collections.singletonList(item))
-                    .show(fragment.getFragmentManager(), "StreamDialogEntry@append_playlist");
+            final PlaylistAppendDialog d = PlaylistAppendDialog
+                    .fromStreamInfoItems(Collections.singletonList(item));
```

Comment:
Maybe extract this code to a function so that it can be reused above

Reply:
I'm not sure what you mean by "extract" in this context. Probably a wrapper for the check- and show-logic? Something like one signature:

`static Disposable safeShow(context, fm, tag, <input>)`

where input is either `StreamInfo` or `List<StreamInfoItem>` or 'List<PlayQueueItem>' (by overriding).

Reply:
Oh, sorry, since the two blocks of code actually do different things you can ignore this suggestion